assets/samples/edgeconnect/automation-workflows.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: dynatrace.com/v1alpha2
+kind: EdgeConnect
+metadata:
+  name: edge-connect-sample
+  namespace: dynatrace
+spec:
+  # Documentation:
+  # https://docs.dynatrace.com/docs/setup-and-configuration/setup-on-k8s/guides/operation/edgeconnect/kubernetes-automation/edge-connect-kubernetes-automation-operator-supported-setup
+
+  apiServer: "https://example.dev.apps.dynatracelabs.com"
+  replicas: 1
+
+  # The service account has to have the proper permissions that match the purpose of the
+  # workflow steps this EdgeConnect is used for.
+  serviceAccountName: edgeconnect-automation
+
+  # Enable K8S Automation for workflows
+  kubernetesAutomation:
+    enabled: true
+
+  oauth:
+    # OAuth client has to be created in Dynatrace account management
+    # https://docs.dynatrace.com/docs/shortlink/account-api-authentication#create-an-oauth2-client
+    #
+    # required scopes:
+    # - app-engine:edge-connects:connect
+    # - app-engine:edge-connects:read
+    # - app-engine:edge-connects:write
+    # - app-engine:edge-connects:delete
+    # - oauth2:client:manage
+    # - settings:objects:read
+    # - settings:objects:write
+    # - settings:schemas:read
```

Comment:
do we really need it? I havent seen it in code, for me it works without this one `settings:schemas:read`